pilot/docker/prepare_proxy.sh

```diff
@@ -1,92 +0,0 @@
-#!/bin/bash
```

Comment:
what's the diff between istio-iptables.sh and this one ?

Reply:
Good question that remained unanswered.  Unfortunately this PR is blocking the docs repo.  A corrective PR is here - but I'm not certain it is correct: https://github.com/istio/istio.github.io/pull/1134

Reply:
This PR replaces istio-iptables.sh with the new version of prepare_proxy.sh that was introduced in https://github.com/istio/istio/pull/4569. This script contains a superset of the functionality of istio-iptables.sh.